Entity/Task.php

```diff
@@ -146,4 +146,86 @@ public function setLog($log)
 
         return $this;
     }
+
```

Comment:
I'n not a big fan of a method for each status.

Reply:
Better readability IMHO